lib/capybara/screenshot/diff.rb

```diff
@@ -14,7 +14,7 @@ module Screenshot
     mattr_accessor :blur_active_element
     mattr_accessor :enabled
     mattr_accessor :hide_caret
-    mattr_reader(:root) { (defined?(Rails.root) && Rails.root) || "." }
+    mattr_reader(:root) { (defined?(Rails.root) && Rails.root) || Pathname(".") }
```

Comment:
Could you make it absolute by `Pathname(".").expand_path`

Reply:
Done.